File: PhysicalTherapy/PhyscialTherapy/Models/Patient.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace PhyscialTherapy.Models
{
    public class Patient
    {
        // maybe area. might have to change db.

        //public string AreaOfFocus { get; set; }

        public string Bio { get; set; }

        public DateTime DateOfBirth { get; set; }

        public string Email { get; set; }

        public string FirstName { get; set; }

        public string LastName { get; set; }

        public int PatientId { get; set; }

        public string PhoneNumber { get; set; }

        public Therapist Therapist { get; set; }

        public int TherapistId { get; set; }

    }
}

File: PhysicalTherapy/PhyscialTherapy/Client/src/app/models/patient.ts
import { Therapist } from "./Therapist";

export interface Patient {
    accountType : number,
    bio : string,
    dateOfBirth : Date,
    email : string,
    firstName : string,
    lastName : string,
    patientId : number,
    phoneNumber : string,
    therapist : Therapist,
    therapistId : number,
    username : string
}